### DISCUSSION TITLE
📢 TheiaCon 2024 is tomorrow!

### DISCUSSION THREAD

**Discussion by JonasHelming** [eclipse-theia/theia#14434] [Show and tell]
📢 **TheiaCon 2024 is tomorrow!**

Get ready for TheiaCon, the annual virtual conference bringing together the Eclipse Theia community! Join us for insightful talks and lightning sessions on November 13–14, 2024. Learn about adopter stories and the latest developments in Theia, Theia IDE, collaboration support and AI-powered tools, and connect with fellow developers and industry experts. Don’t miss out!

🔗 [Learn more and register here](https://www.eclipse.org/events/2024/theiacon/)

**Comment by cbries** [comment]
Do you record the sessions and will you publish them somewhere?
Because of family the timeslots do not really fit for me.

**Reply by JonasHelming** [reply]
Sessions will be recorded and will be made available. Only the ones who have good excuses like yo may consume them though :-)
